EIPS/eip-4353.md

```diff
@@ -0,0 +1,230 @@
+
+#### Constructor/deployment
+* Creator - the owner of an NFT with its own rules for depositing tokens at and/or after the minting of a token.
+* Token Amount - the current amount of on-chain [EIP-20](./eip-20.md) or derived tokens bound to an NFT from one or more deposits.
+* Withdraw Mechanism - rules based approach for withdrawing staked tokens and making sure to update the balance of the staked tokens.
+
+### Staking at mint and locking tokens in NFT
+The suggested and intended implementation of this standard is to stake tokens at the time of minting an NFT, and not implementing any outbound transfer of tokens outside of `burn`. Therefore, only to stake at minting and withdraw only at burning.
+
+#### NFT displayed in wallet or marketplace
+A wallet or marketplace checks if an NFT has publicly staked tokens available for display - if so, call `stakedAmount(tokenId)` to get the current amount of tokens staked and/or bound to the NFT.
+
+The logical code looks something like this and inspired by William Entriken:
+
+```solidity
+// contracts/Token.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Token
+ * @dev Very simple ERC721 example with stake interface example.
+ * Note this implementation enforces recommended procedure:
+ *  1) stake at mint
+ *  2) withdraw at burn
+ */
+contract ERC721Staked is ERC721URIStorage, Ownable {
+    /// @dev track original minter of tokenId
+    mapping (uint256 => address payable) private payees;
+    /// @dev map tokens to stored staked token value
+    mapping (uint256 => uint256) private tokenValue;
+
+    /// @dev metadata
+    constructor() ERC721 (
+        "Staked NFT", 
+        "SNFT"
+    ){}
+
+    /// @dev mints a new NFT
+    /// @param _to address that will own the minted NFT
+    /// @param _tokenId id the NFT
+    /// @param _uri metadata
+    function mint(
+        address payable _to,
+        uint256 _tokenId,
+        string calldata _uri
+    )
+        external 
+        payable
+        onlyOwner
+    {
+        _mint(_to, _tokenId);
+        _setTokenURI(_tokenId, _uri);
+        payees[_tokenId] = _to;
+        tokenValue[_tokenId] = msg.value;
+    }
+
+    /// @dev staked interface
+    /// @param _tokenId id of the NFT
+    /// @return _value staked value
+    function stakedAmount(
+        uint256 _tokenId
+    ) external view returns (uint256 _value) {
+        _value = tokenValue[_tokenId];
+        return _value;
+    }
+
+    /// @dev removes NFT & transfers crypto to minter
+    /// @param _tokenId the NFT we want to remove
+    function burn(
+        uint256 _tokenId
+    )
+        external
+        onlyOwner
+    {
+        super._burn(_tokenId);
+        payees[_tokenId].transfer(tokenValue[_tokenId]);
+        tokenValue[_tokenId] = 0;
+    }
+
+}
+```
+
+## Backward Compatibility
+TBD
```

Comment:
I must have been completely asleep when I wrote my last comment. I'll open a PR to fix my own dumb mistake ^^;;